packages/core/src/entityConfiguration/entityConfigurationClaims.ts

```diff
@@ -6,4 +6,8 @@ export const entityConfigurationClaimsSchema = entityStatementClaimsSchema.refin
   path: ['iss', 'sub'],
 })
 
-export type EntityConfigurationClaims = z.input<typeof entityConfigurationClaimsSchema>
+type EntityConfigurationClaimsInput = z.input<typeof entityConfigurationClaimsSchema>
+
+type EntityConfigurationClaimsOutput = z.output<typeof entityConfigurationClaimsSchema>
+
+export type EntityConfigurationClaims = EntityConfigurationClaimsInput | EntityConfigurationClaimsOutput
```

Comment:
I wouldn't type it like this, as it's weird to use this type. We can't easily detect whether we have input or output, so this will complicate things.

You either expect the output type (transformed) or the input type (which is unvalidated/not parsed and will have to go through zod before using)